litert/vendors/qualcomm/core/wrappers/op_wrapper.cc

```diff
@@ -79,4 +95,18 @@ Qnn_OpConfig_t OpWrapper::GetOpConfig() {
   return qnn_op;
 }
 
+const char* OpWrapper::GetTypeName() const { return type_name_; }
```

Comment:
I saw most use cases of `GetTypeName()` are comparing with `QNN_OP_XXX`, we can change this function into something like:
```suggestion
bool OpWrapper::IsOpType(const char *op_type) const { return strcmp(type_name_, op_type); }
```

Reply:
Changed to `IsOpType`. Thank you!